libbeat/docs/processors-using.asciidoc

```diff
@@ -169,18 +169,22 @@ endif::[]
 The supported processors are:
 
  * <<add-cloud-metadata,`add_cloud_metadata`>>
+ * <<add-docker-metadata,`add_docker_metadata`>>
+ * <<add-fields, `add_fields`>>
+ * <<add-host-metadata,`add_host_metadata`>>
+ * <<add-kubernetes-metadata,`add_kubernetes_metadata`>>
+ * <<add-labels, `add_labels`>>
  * <<add-locale,`add_locale`>>
+ * <<add-process-metadata,`add_process_metadata`>>
+ * <<add-tags, `add_tags`>>
+ * <<community-id,`community_id`>>
```

Comment:
This line needs to be deleted. Results in an error: "element link: validity error : IDREF attribute linkend references an unknown ID "community-id""